examples/LL87.xml

```diff
@@ -221,6 +221,11 @@
             </ScenarioType>
           </Scenario>
         </Scenarios>
+        <Qualifications>
+          <Qualification ID="ReportQualification-01">
```

Comment:
I'm not really buying the need for this. I can't think of a situation in which this is going to be needed internally, but I won't hold things up over it since it isn't required.